src/components/EventCard.tsx

```diff
@@ -122,39 +134,41 @@ export const EventCard = ({ data }: EventCardProps) => {
           >
             {description}
           </Typography>
-          <Button
-            variant="contained"
-            disableElevation
-            color="primary"
-            href={link.uri}
-            target="_blank"
-            endIcon={
-              <LaunchIcon sx={{ marginLeft: '-4px', width: 16, height: 16 }} />
-            }
-            sx={{
-              backgroundColor: 'white',
-              color: '#226488',
-              fontWeight: 500,
-              lineHeight: 1.4,
-              cursor: 'pointer',
-              padding: '10px 12px',
-              border: `1px solid #226488`,
-              borderRadius: 8,
-              '&:hover': {
-                backgroundColor: 'white',
-              },
-              '&:focus': {
-                outline: 'none',
-              },
-              '&.MuiButton-root': {
-                boxShadow: 'none',
-              },
-            }}
-          >
-            {link.label}
-          </Button>
         </Box>
-      </Grid>
+      </Box>
+      <Box sx={{ marginTop: 'auto' }}>
+        <Button
+          variant="contained"
+          disableElevation
+          color="primary"
+          href={link.uri}
+          target="_blank"
```

Comment:
check if is intern link or external to decided for the target.

Reply:
It looks like an external link since it is a meetup event link.

Reply:
it should not be hardcoded, because it is an api and it can return anything that is what I mean. @stepsen89 I remember that was something that NexJS would take care, could you check here if it is correct?

Reply:
Will these events always have a meet up link? Where would it link within the application (we don't have an `event/{id}` page for a detail of the event). If you think we will have internal paths too, we can add a check 👍 
We discussed this component today and are looking into using Link instead of Button too